src/main/java/com/marksem/crm/facade/DocumentFacade.java

```diff
@@ -0,0 +1,71 @@
+package com.marksem.crm.facade;
+
+import com.marksem.crm.contract.DefaultIml;
+import com.marksem.crm.contract.DocumentImp;
+import com.marksem.crm.dto.request.DocumentDtoRequest;
+import com.marksem.crm.dto.response.DocumentDtoResponse;
+import com.marksem.crm.entity.Document;
+import com.marksem.crm.service.DocumentService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Component
+public class DocumentFacade implements DefaultIml<DocumentDtoRequest, DocumentDtoResponse>, DocumentImp<DocumentDtoRequest, DocumentDtoResponse> {
```

Comment:
Try making part of this file abstract and generic - you will need this code for all entities